x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -479,7 +479,7 @@ export function removeLayer(id) {
   };
 }
 
-export function setQuery({ query, timeFilters }) {
+export function setQuery({ query, timeFilters, filters = [] }) {
```

Comment:
Should we consider consolidating timeFilters and filters under a top-level filters object? Or if we're after a flatter hierarchy, going in the other direction, separating out other different filters by name? Could be just a name change would suffice here too.

Reply:
this is fine imo. we already collect it in mapState. it's readable.